--- exiv2-0.11/src/Makefile.no_rpath	2006-09-19 13:49:19.000000000 -0500
+++ exiv2-0.11/src/Makefile	2006-09-19 13:50:05.000000000 -0500
@@ -196,7 +196,7 @@
 COMPILE.cc = $(CXX) $(CXXFLAGS) $(CXXDEFS) $(CXXINCS) -c
 COMPILE.c = $(CC) $(CFLAGS) $(DEFS) $(INCS) -c
 # LINK.cc does not need $(LIBS), libtool's dark magic takes care of that
-LINK.cc = $(CXX) $(CXXFLAGS) $(LIBRARY) $(LDFLAGS) -rpath $(libdir)
+LINK.cc = $(CXX) $(CXXFLAGS) $(LIBRARY) $(LDFLAGS) #-rpath $(libdir)
 
 # ******************************************************************************
 # Rules
